data/basal.go

```diff
@@ -0,0 +1,141 @@
+package data
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	valid "github.com/tidepool-org/platform/Godeps/_workspace/src/gopkg.in/bluesuncorp/validator.v8"
```

Comment:
Have you had a chance to clean up with `validator` global variable into a Getter and then be able to get rid of the special `valid` import here? (From the previous PR.)